.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false,
```

Comment:
I think you forgot to change this one 😛

Reply:
Ah sorry, i missunderstood that we actually want to try semi-less aproach 😅